toolchains/rbe_toolchains_config.bzl

```diff
@@ -96,7 +96,7 @@ def _envoy_rbe_toolchain(name, env, toolchain_config_spec_name, toolchain_config
     )
 
 def rbe_toolchains_config(generator = False, force = False):
-    linux_exec_properties = create_rbe_exec_properties_dict(docker_add_capabilities = "SYS_PTRACE,NET_RAW,NET_ADMIN", docker_network = "standard")
+    linux_exec_properties = create_rbe_exec_properties_dict(docker_add_capabilities = "SYS_PTRACE,NET_RAW,NET_ADMIN", docker_network = "standard", docker_ulimits="memlock=-1,nice=-20,rtprio=10,stack=8388608")
```

Comment:
What is the current stack config for the RBE toolchain on docker?  I think I have been able to repo the long argument list problem when building with gcc locally despite runs of `ulimit -s` on my machine returning 8192